### PR TITLE
Build script for envy golang package

### PR DESCRIPTION
**Pull request by ezeeyahoo**
Hi,

Please review and provide your feedback.

Thanks

### CODE REVIEW

File: e/envy/envy_ubuntu_20.04.sh [hunk 1]
```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+# ----------------------------------------------------------------------------
+#
+# Package        : Envy
+# Version        : 1.7.1
```
Comment (gururajrkatti) [inline e/envy/envy_ubuntu_20.04.sh:5]: Are we not testing the latest version ?

File: e/envy/envy_ubuntu_20.04.sh [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+# Source repo    : https://github.com/gobuffalo/envy
+# Tested on      : Ubuntu 20.04 + Go latest version (v1.7.1)
```
Comment (gururajrkatti) [inline e/envy/envy_ubuntu_20.04.sh:7]: As per the requirement we need to test on ubi 8, is there any issue on that platform ?

File: e/envy/envy_ubuntu_20.04.sh [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+# Script License : GNU Lesser General Public License Version 2.1
```
Comment (gururajrkatti) [inline e/envy/envy_ubuntu_20.04.sh:8]: This should be the apache license, as per other scripts in the same folder.

File: e/envy/envy_ubuntu_20.04.sh [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    echo "Setting up GOPATH"
+    echo "export GOPATH=$(pwd)/go" >> .profile && source .profile
+
+    echo "Setting PATH to include golang binaries"
+    echo "export PATH='$PATH':$(pwd)/golang/go/bin:$GOPATH/bin" >> .profile && source ~/.profile
+
+fi
+
+# install envy package
+go install github.com/gobuffalo/envy@v1.7.1
```
Comment (gururajrkatti) [inline e/envy/envy_ubuntu_20.04.sh:51]: Will it run the unit tests ? Are we not passing a command line argument to accept any version and test ?

**Review by gururajrkatti — CHANGES_REQUESTED** [review]

File: e/envy/LICENSE [hunk 5]
```diff
@@ -0,0 +1,201 @@
+Apache License
```
Comment (gururajrkatti) [inline e/envy/LICENSE:1]: Indentation issue

File: e/envy/envy_ubi8.3.sh [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+# Script License : Apache 2.0
+# Maintainer     : Eshant Gupta <eshant.gupta@ibm.com>; Priya Seth <sethp@us.ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# Usage: ./envy_ubi_8.3.sh
```
Comment (gururajrkatti) [inline e/envy/envy_ubi8.3.sh:17]: Pl. dont modify the format of header

File: e/envy/envy_ubi8.3.sh [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+#                  OR
+# 	 ./envy_ubi_8.3.sh v1.7.1
+# ----------------------------------------------------------------------------
+
+source ~/.profile
```
Comment (gururajrkatti) [inline e/envy/envy_ubi8.3.sh:22]: Is it required ?

File: e/envy/envy_ubi8.3.sh [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+
+if [ "$#" -gt 1	]
+then
+	usage "$(basename $0)"
+else
+	[ -n "$1" ] && export TAGS="$1" || export TAGS=""
+fi
+
+# check installed Golang
+[ -n $(which go) ] && [[ $(go version) > 1.6 ]] && set_env=true
```
Comment (gururajrkatti) [inline e/envy/envy_ubi8.3.sh:41]: Isn't the latest version 1.17 ?

File: e/envy/envy_ubi8.3.sh (hunk 6 above)
Comment (ezeeyahoo) [reply e/envy/envy_ubi8.3.sh:17]: Shall I remove Usage comment?

File: e/envy/envy_ubi8.3.sh (hunk 8 above)
Comment (ezeeyahoo) [reply e/envy/envy_ubi8.3.sh:41]: typo fixed, it has to be 1.16. if version is not found or found to be earlier release, then script download the latest go version.

**Pull request merged**